standards/bulk.md

```diff
@@ -0,0 +1,174 @@
+
+### Synchronous
+
+Bulk operations **MUST** be synchronous when applied to an existing resource. This means that the client will receive a response to the bulk request only after all operations have been completed. This is in contrast to asynchronous operations, where the client receives a response immediately after the request is received, and then must poll for the result of the operation. Asynchronous bulk operations are applied to new resources specifically for bulk or import to enable subsequent status updates through additional endpoints.
+
+- Bulk operations **MUST** be specific to a single resource type and NOT allow for updating multiple resource types in a single request.
+- Bulk operations **MUST** be implemented as a PATCH request against a collection resource that not idempotent.
+- Bulk operations **MUST** return a `200 (OK)` response code if all operations were received and a result is available for each operation. Bulk operations **MUST NOT** use status code `207 (Multi-Status)` response code as this incurs other implications for the response schema in relationship to WebDAV. System level errors may still result in a `500 (Internal Server Error)` response code where the request could not be processed or was prevented from trying operations.
+- Bulk operations **MUST** accept a constrained number of operations in the request body that is indicated in the documentation of the endpoint. By default this value **MAY** be 100 operations, but should be adjusted according to the needs of the endpoint and the entity-size. Requests beyond the limit **MUST** return a `400` response code and standard [error format body](errors.md) similar to:
+```json
+// RESPONSE
+HTTP/1.1 400
+Content-Type: application/problem+json
+{
+    "title": "Invalid Data",
+    "status": 400,
+    "detail": "Operations collection may only contain a maximum of '100' actions per request.",
+    "instance": "/articles",
+    "requestId": "b6d9a290-9f20-465b-bcd3-4a5166eeb3d7"
+}
+```
+
+#### Request
+
+- Bulk operations **MUST** include a request body schema 
+```json
+{
+    "transaction": "ATOMIC" | "ISOLATED",                                   // default: "ATOMIC", optional. transactionality of the request
```

Comment:
Formatting of this looks a bit strange 🤷‍♂️.

Default value is also in conflict with the paragraph below.
```suggestion
    "transaction": "ATOMIC" | "ISOLATED",                                   // optional transactionality of the request, default: "ISOLATED"
```

Reply:
nice! good callout.